packages/compose/src/hooks/use-reduced-motion/index.js

```diff
@@ -3,13 +3,20 @@
  */
 import useMediaQuery from '../use-media-query';
 
+/**
+ * Whether or not the user agent is Internet Explorer.
+ *
+ * @type {boolean}
+ */
+const IS_IE = window.navigator.userAgent.indexOf( 'Trident' ) >= 0;
```

Comment:
Just a heads up that this line breaks the mobile app.
I created a PR on gutenberg-mobile side to solve it: https://github.com/wordpress-mobile/gutenberg-mobile/pull/1195

![IMG_2271](https://user-images.githubusercontent.com/9772967/60679189-144ee880-9e87-11e9-9b54-fcd4475cde7d.PNG)

Reply:
Oh sorry about that. Should we maybe create a custom version of `useReducedMotion` that always return "true" on mobile? Is there an equivalent to `prefers-reduced-motion` in mobile?

Reply:
I think the equivalent landed on 0.60 which was just released: https://github.com/facebook/react-native/pull/23839

Reply:
Nice, let's add the equivalent hook once we upgrade to 0.60